src/library/Uno.Themes.WinUI.Markup/ResourceValue.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using Microsoft.UI.Xaml.MarkupHelpers;
+
+namespace Uno.Themes.Markup
+{
+	public struct ResourceValue<T>
+	{
+		public ResourceValue(string key)
+		{
+			Key = key;
+		}
+
+		public string Key { get; }
+
+		public static implicit operator Action<IDependencyPropertyBuilder<T>>(ResourceValue<T> resource) =>
+			x => x.ThemeResource(resource.Key);
```

Comment:
Ah I forgot about the ThemeResource/StaticResource usage. I believe we should be using `x.ThemeResource` for Brushes and Colors and `x.StaticResource` for Styles. Maybe we should add a `bool isThemeResource` to the `ResourceKey` constructor?

Reply:
Sure, I'm adding it and I'll create a new PR. I made a mistake thinking about `x.StaticResource` and `X.ThemeResource` are the same and I didn't ask you. Sorry for that.